new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful and popular editor that has many commands. Unlike most
+text editors where you use the keyboard to enter text and arrow keys to move you
+cursor, Vim uses the keyboard to enter commands to manipulate the text.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these commands
+are not arbitrarily chosen, they form a language with its own grammar.
```

Comment:
If the tutorial did not teach you the basics of the language it would be kind of pointless, wouldn't it? I think that should be expected and doesn't need mentioning.